src/deck-components/SidebarNavigation.tsx

```diff
@@ -16,7 +16,7 @@ export interface SidebarNavigationPage {
 
 export interface SidebarNavigationProps {
   title?: string;
-  pages: SidebarNavigationPage[];
+  pages: (SidebarNavigationPage|string)[];
```

Comment:
Since "separator" is the only valid string (atleast that we know of) I think leaving it as a blanket `string[]` is a little dangerous as it won't correctly flag invalid strings as errors.

Change `(SidebarNavigationPage|string)[]` to `(SidebarNavigationPage|"separator")[]`